app/presenters/education_navigation_ab_testable.rb

```diff
@@ -0,0 +1,17 @@
+module EducationNavigationABTestable
```

Comment:
An alternative implementation would be to make this a class that you instantiate with a content item and a request object. You'd then create an instance of it in ApplicationController#breadcrumbs:

```
ab_testing_policy = EducationNavigationABTestingPolicy.new(@content_item, request)
...
return @navigation_helpers.taxon_breadcrumbs if ab_testing_policy.should_present_new_navigation_view?
return @navigation_helpers.breadcrumbs
...
```

Given none of this logic is doing any actual presenting, a better home for it might be `app/models`.

Benefits: 
* The resulting class is simpler to unit test.
* Its dependencies are obvious within the first few lines of its implementation.
* You can mark the latter three methods private, hinting they aren't used outside the class.
* Renaming and relocating the file makes its purpose a little clearer/easier to remember without needing to read the implementation.

Better still would be to extract out the entire notion of a NavigationPresenter from the controller layer and letting it handle breadcrumbs and related links, but that's probably a bit much for one PR.